app/furniture/marketplace/management_component.html.erb

```diff
@@ -21,6 +21,7 @@
         <%= render button({child: :notification_methods}, icon: :bell)  if policy(marketplace.notification_methods).index? %>
         <%= render button({child: :flyer}, icon: :receipt_percent)  if policy(marketplace.flyer).show? %>
         <%= render button({child: :vendor_representatives}, icon: :cake) if policy(marketplace.vendor_representatives).index? %>
+        <%= render button({child: :tags}, icon: :tag) if policy(marketplace.tags).index? %>
```

Comment:
This uses the same icon as `Marketplace#products`, so maybe we should consider getting a different icon for `Product`? Maybe something food-like?

Reply:
https://github.com/zinc-collective/convene/pull/2221